prompb/types.proto

```diff
@@ -56,3 +57,25 @@ message ReadHints {
   int64 start_ms = 3; // Start time in milliseconds.
   int64 end_ms = 4;   // End time in milliseconds.
 }
+
+// Chunk represents a TSDB chunk.
+message Chunk {
+  int64 min_time_ms = 1;
+  int64 max_time_ms = 2;
+
+  // We require this to match chunkenc.Encoding.
+  enum Encoding {
+    UNKNOWN = 0;
+    XOR     = 1;
+  }
+  Encoding type  = 3;
+  bytes data     = 4;
+}
+
+// ChunkedSeries represents single, encoded time series.
+message ChunkedSeries {
+  // labels should be sorted.
+  repeated Label labels = 1 [(gogoproto.nullable) = false];
+  // chunks should be sorted should not overlap in time.
```

Comment:
Does this work with backfill? 

Also, Sentence case on comments please.

Reply:
That's a case where we'd have to reencode I suspect.

Reply:
Why?  

We could relax this constraint to something like "chunks will be in start time order and may overlap." and leave merging to the client.

Reply:
That's extra complexity for every client to handle, and they'll likely only do so when the bug reports come in.